pkg/testutil/require.go

```diff
@@ -67,7 +66,7 @@ func RequireClusterToBeReadyEventuallyTimeout(t *testing.T, sb testenv.DiffingSa
 
 		if !statefulSetIsReady(ss) {
 			t.Logf("stateful set is not ready")
-			logPods(context.TODO(), ss, cluster, sb, t)
+			_ = logPods(context.TODO(), ss, cluster, sb, t)
```

Comment:
if logPods can return an error, maybe we should t.Logf it?

Reply:
Done